### ISSUE TITLE
Copy&Paste Order in Layout view improvement

### ISSUE THREAD

**Issue by A380Coding** [FlowFuse/node-red-dashboard#1528]
### Description

In Dashboad 2.0 -> Layout re-ordering and moving of widgets, groups, pages is possible. In case of copy&paste of already existing elements, the order is in a strange way. Some nodes get added at the bottom of the tree view, sometimes some in the middle.
Also all Tree elements get expanded when working on items.

It is not clear which action triggers this behavior. But it is difficult to work with. In Dashboard 1 compacted list elements stayed compacted when working on other list elements. When copying elements, the copy of each element was entered below the source.

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by bartbutenaers** [comment]
@A380Coding,

> Also all Tree elements get expanded when working on items.

Yes I have to admit that it is kind of hard to work with the sidebar, because it is being refreshed automatically.  
I am going to add a link here to another [issue](https://github.com/FlowFuse/node-red-dashboard/issues/1340) that already reported that misbehaviour some time ago.
